pkg/proxy/apis/kubeproxyconfig/v1alpha1/types.go

```diff
@@ -22,15 +22,15 @@ import (
 
 // ClientConnectionConfiguration contains details for constructing a client.
 type ClientConnectionConfiguration struct {
-	// kubeConfigFile is the path to a kubeconfig file.
+	// kubeconfig is the path to a kubeconfig file.
 	KubeConfigFile string `json:"kubeconfig"`
 	// acceptContentTypes defines the Accept header sent by clients when connecting to a server, overriding the
 	// default value of 'application/json'. This field will control all connections to the server used by a particular
 	// client.
 	AcceptContentTypes string `json:"acceptContentTypes"`
 	// contentType is the content type used when sending data to the server from this client.
 	ContentType string `json:"contentType"`
-	// cps controls the number of queries per second allowed for this connection.
+	// qps controls the number of queries per second allowed for this connection.
```

Comment:
QPS (capitalized)

Reply:
At one point, the first word of the comment needed to match the json tag. I'm following that convention. Not sure if it's still applicable. Also, my pending (possibly dead) #52198 relies on the first word matching the json tag name.

@liggitt care to comment?

Reply:
We match JSON names now.  It was a mistake.  One we live with for now.